src/components/CardInputs/reps.js

```diff
@@ -0,0 +1,13 @@
+export default function CardInputReps({reps, handleRepInput}) {
+	return (
+		<label>
+			Reps:{' '}
```

Comment:
You can delete the `{ '  ' }`
```suggestion
			Reps:
```